Code/Framework/AzCore/AzCore/Settings/SettingsRegistryImpl.cpp

```diff
@@ -799,17 +816,23 @@ namespace AZ
 
         SignalNotifier(anchorKey, anchorType);
 
-        return true;
+        MergeSettingsResult mergeResult;
+        mergeResult.m_returnCode = MergeSettingsReturnCode::Success;
+        return mergeResult;
```

Comment:
not suggesting this for this PR but what do you think about the struct initializer syntax in the future for things like this? e.g.
```
return { .m_returnCode = MergeSettingsReturnCode::Success };
```

Reply:
Well I would like to use that syntax, but that relies on us compiling the codebase with C++20 which we do not at the time